src/components/pages/clidocs/OverviewPage.vue

```diff
@@ -0,0 +1,233 @@
+<template>
+    <v-container>
+        <h1 class="font-weight-light">
+            Unipept <Initialism>CLI</Initialism> documentation <small>v2.2.1</small>
+        </h1>
+        <h3 class="font-weight-light">
+            The Unipept command line interface (<Initialism>CLI</Initialism>) offers an easy way to integrate Unipept metaproteomics functionality into your data 
+            processing pipelines and scripts.
+        </h3>
+
+        <p class="mt-5">
+            These tools provide a command line interface to the <RLink to="/apidocs" router>Unipept web services</RLink> and a few utility commands for handling proteins using the command line. All tools support fasta and plain text input, multiple output formats (csv, xml and json) and parallel web requests for improved performance.
+        </p>
+
+        <p class="mt-3">
+            The Unipept command line tools are open source under the <Initialism>MIT</Initialism> License and all code is available on 
+            <RLink to="https://github.com/unipept/unipept-cli">Github</RLink>. Changes between releases are listed in the <RLink to="/news" router>changelog</RLink>. 
```

Comment:
I think it would be better if we directly link to the changelogs of the CLI itself (and not the overview of the changes of all Unipept tools) for the "changelog"-link here.